pkgs/applications/networking/instant-messengers/mikutter/update.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, pname
+, version
+, writers
+, bundix
+, bundler
+, common-updater-scripts
+, curl
+, jq
+}:
+
+let
+  binPath = lib.makeBinPath [ bundler bundix curl jq common-updater-scripts ];
+in
+writers.writeBash "update-mikutter" ''
```

Comment:
Why are you not using a script wit nix-shell?

Reply:
Mostly because it's easy to embed Nix variables. I've now changed it to a nix-shell script instead.